src/coord/src/catalog/table_updates.rs

```diff
@@ -0,0 +1,399 @@
+// Copyright Materialize, Inc. All rights reserved.
```

Comment:
nit: maybe rename the file to `builtin_table_updates.rs`?

Reply:
Ah yeah I went back and forth on this! You've broken the tie so I've renamed accordingly!